### ISSUE TITLE
Accessibility Improvements

### ISSUE THREAD

**Issue by lewisgoddard** [BanhamPoultryUK/website#1]
- [ ] Navigation menu fallback color
- [ ] Linked images missing alternative text
- [ ] No heading structure
- [ ] Links are not descriptive
- [ ] PDF Documents
- [ ] Null or empty alternative text
- [ ] Content IDs are inconsistent
